custom-tests.yaml

```diff
@@ -1517,6 +1515,10 @@ api:
   RadioNodeList:
     __base: >-
       <%api.HTMLFormControlsCollection:controls%>
+      for (var i = 0; i < controls.length; i++) {
```

Comment:
This will result in two loops in the generated test. I was thinking rather starting from `<%api.HTMLFormElement:form%>` and just duplicating some of the code. Simplicity of the generated tests is more important than code reuse internally.